gpMgmt/bin/gppylib/commands/unix.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+        #!/usr/bin/env python3
```

Comment:
Nit: unnecessary indention?

```suggestion
#!/usr/bin/env python3
```